fiftyone/core/session/events.py

```diff
@@ -159,11 +159,18 @@ class CustomizeColor:
     valueColors: t.Optional[t.List[ValueColor]] = None
 
 
+@dataclass
+class LabelTagColor:
+    fieldColor: t.Optional[str] = None
```

Comment:
Maybe `defaultColor` is more informative? Label tags apply to all label fields

Reply:
I prefer `fieldColor`, as it acts the same way as `fieldColor` in `ColorScheme.fields` as a selected color for the field. I would associate `defaultColor` with the color generated from `getColor(pool, seed, "_label_tags")`.